app-java/src/main/java/ly/count/java/demo/Sample.java

```diff
@@ -132,7 +132,7 @@ public void LogHappened(String logMessage, Config.LoggingLevel logLevel) {
                     //System.out.println("[" + logLevel + "] " + logMessage);
                 }
             })
-            .setEventsBufferSize(1)//this should be only used for testing
+            .setEventQueueSizeToSend(1)//this should be only used for testing
```

Comment:
this comment still holds?

Reply:
I guess the following would be a better message:
'setting queue size to "1" should only be done for testing, unless you have a really good reason to do it'